guides/source/configuring.md

```diff
@@ -3336,7 +3336,7 @@ Now the behavior is clear, that we are only using the connection information in
 
 #### Configuring an SQLite3 Database
 
-Rails comes with built-in support for [SQLite3](http://www.sqlite.org), which is a lightweight serverless database application. While a busy production environment may overload SQLite, it works well for development and testing. Rails defaults to using an SQLite database when creating a new project, but you can always change it later.
+Rails comes with built-in support for [SQLite3](http://www.sqlite.org), which is a lightweight serverless database application. In Rails 7.1 and beyond, SQLite is better configured for production workloads, but a busy production environment may overload SQLite. Rails defaults to using an SQLite database when creating a new project, but you can always change it later.
```

Comment:
This documentation is for Rails 8.0, there is no point to talk about versions before it that might not even be maintained anymore. Let's just remove any mention to versions.

```suggestion
Rails comes with built-in support for [SQLite3](http://www.sqlite.org), which is a lightweight serverless database application. While Rails, better configures SQLite for production workloads, a busy production environment may overload SQLite. Rails defaults to using an SQLite database when creating a new project, but you can always change it later.
```

Reply:
Good point. Dropped the hanging comma, but made this change. Thank you